cmake/macros/macro_configure_feature.cmake

```diff
@@ -233,10 +233,14 @@ macro(CONFIGURE_FEATURE _feature)
         #
         # Second case: We are allowed to search for an external library
         #
-        if(COMMAND FEATURE_${_feature}_FIND_EXTERNAL)
-          evaluate_expression("FEATURE_${_feature}_FIND_EXTERNAL(FEATURE_${_feature}_EXTERNAL_FOUND)")
+        if(NOT FEATURE_${_feature}_EXTERNAL_FOUND AND NOT DEAL_II_${_feature}_FOUND)
+          if(COMMAND FEATURE_${_feature}_FIND_EXTERNAL)
+            evaluate_expression("FEATURE_${_feature}_FIND_EXTERNAL(FEATURE_${_feature}_EXTERNAL_FOUND)")
+          else()
+            feature_find_external(${_feature} FEATURE_${_feature}_EXTERNAL_FOUND)
+          endif()
         else()
-          feature_find_external(${_feature} FEATURE_${_feature}_EXTERNAL_FOUND)
+          set(FEATURE_${_feature}_EXTERNAL_FOUND TRUE)
         endif()
```

Comment:
This PR also contains this small bugfix so that it is actually possible again to override the entire setup of an external dependency via command line...